samples/drivers/jesd216/src/main.c

```diff
@@ -241,8 +241,13 @@ void main(void)
 	} u;
 	const struct jesd216_sfdp_header *hp = &u.sfdp;
 	int rc = flash_sfdp_read(dev, 0, u.raw, sizeof(u.raw));
-	uint32_t magic = jesd216_sfdp_magic(hp);
+	if (rc != 0) {
+		printf("Read SFDP not supported: device not JESD216-compliant "
+		       "(err %d)\n", rc);
+		return;
+	}
 
+	uint32_t magic = jesd216_sfdp_magic(hp);
```

Comment:
yeap, you need a blank line, here as far as I remember

Reply:
@ioannisg Added.